htdocs/compta/bank/categ.php

```diff
@@ -118,44 +89,28 @@
 print '<td>'.$langs->trans("Ref").'</td><td colspan="2">'.$langs->trans("Label").'</td>';
 print "</tr>\n";
 
-$sql = "SELECT rowid, label";
```

Comment:
But why not modifying fetchAll function to support that when that feature is developed? The goal should be to avoid using queries in the frontend…